firebase.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
If you're curious about what this file does exactly, check [these docs](https://firebase.google.com/docs/hosting/full-config) out.